elastic/kibana
# [Alerting] Add telemetry to task manager health API to determine what users are accessing it

Relates to https://github.com/elastic/kibana/issues/102100

## Summary

We are able to limit access to the task manager health API to users with Global All privilege but we want to be cautious about adding this restriction since we currently ask for the health output frequently while debugging issues in SDHs. As a step toward adding this restriction, this PR adds a usage counter to determine the number of times the API is accessed vs the number of time the API is accessed by an admin user.

## To Verify
* Access `api/task_manager/_health` as a super user. Verify that this access is captured in a usage counter by checking  `/api/stats?extended=true&legacy=true`
* Create a role with less than superuser access. Create a user for that role.
* Access `api/task_manager/_health` as this new user. Verify that this access is captured in a usage counter by checking  `/api/stats?extended=true&legacy=true`

### Checklist

Delete any items that are not applicable to this PR.
